app/api/validations/organization.py

```diff
@@ -27,15 +29,21 @@ def validate_update_organization(data):
         return messages.ORGANIZATION_STATUS_INPUT_IS_INVALID
     except KeyError:
         return messages.ORGANIZATION_OR_PROGRAM_STATUS_FIELD_IS_MISSING
-    
-    
+
+
 def validate_update_program(data):
+    if "program_name" not in data:
+        return PROGRAM_NAME_IS_MISSING
```

Comment:
remove explicit import and add ` return messages.PROGRAM_NAME_IS_MISSING` and `return messages.PROGRAM_NAME_ALREADY_USED`